db/migrate/20221102045508_add_encrypt_pms_default_to_user_options.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddEncryptPmsDefaultToUserOptions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_options, :encrypt_pms_default, :boolean, default: default_value, null: false
```

Comment:
I think rather than setting a default for this column it would be best to backfill instead, because the default value will not reflect `encrypt_pms_default` if that gets changed, and we are already automatically setting this on the model callback as well. So it would be

1. add column that is nullable
2. backfill encrypt_pms_default to reflect the site setting
3. do a post_migrate that does a second backfill and changes the column to NOT NULL

Reply:
I think we don't need both post-migration and second backfill.